concepts/05 UI Components/Diagram/30 Diagram Tools.md

```diff
@@ -137,10 +137,11 @@ The Diagram UI component allows you to customize its UI tools with the following
       </dx-diagram>
 
       <!-- tab: app.component.ts -->
+      import { confirm } from 'devextreme/ui/dialog';
       // ...
       onCustomCommand(e) {
```

Comment:
The function should be declared inside a class. I think we need to show the full code.